src/routes/proxyLinkPreview.route.ts

```diff
@@ -0,0 +1,17 @@
+import express, { Request, Response } from "express";
+
+import validUrl from "../helpers/validUrl";
+import { getMetadata } from "../services/proxyLinkPreview.service";
+
+const proxyLinkPreviewRouter = express.Router();
+
+proxyLinkPreviewRouter.route("/").get(async (req: Request, res: Response) => {
```

Comment:
```suggestion
proxyLinkPreviewRouter.route("/").get(async (req: Request<{ url: string }>, res: Response) => {
```

if I'm not wrong you can type query like this

Reply:
url in query.